modules/artifact-activation/src/main/java/org/mule/runtime/module/artifact/activation/internal/nativelib/NativeLibraryFinder.java

```diff
@@ -20,4 +22,11 @@ public interface NativeLibraryFinder {
    * @return library path to use for the given name. Can be null is no library was found.
    */
   String findLibrary(String name, String libraryPath);
+
+  /**
+   * Provides the shared native libraries
+   *
+   * @return the shared native libraries. Can be an empty list if no native library is found.
+   */
+  List<String> getNativeLibraries();
```

Comment:
Add a TODO that will be deleted after the same tasks that we added to the ApplicationClassloader.